consensus-types/light-client/finality_update.go

```diff
@@ -92,18 +92,22 @@ func NewFinalityUpdateFromUpdate(update interfaces.LightClientUpdate) (interface
 // In addition to the proto object being wrapped, we store some fields that have to be
 // constructed from the proto, so that we don't have to reconstruct them every time
 // in getters.
-type FinalityUpdateAltair struct {
+type finalityUpdateAltair struct {
 	p               *pb.LightClientFinalityUpdateAltair
 	attestedHeader  interfaces.LightClientHeader
 	finalizedHeader interfaces.LightClientHeader
 	finalityBranch  interfaces.LightClientFinalityBranch
 }
 
-func (u *FinalityUpdateAltair) IsNil() bool {
+func NewEmptyFinalityUpdateAltair() interfaces.LightClientFinalityUpdate {
```

Comment:
Can you move this constructor up, right after the previous constructor? Same for others